test/e2e/lifecycle/reboot.go

```diff
@@ -196,7 +195,7 @@ func printStatusAndLogsForNotReadyPods(c clientset.Interface, ns string, podName
 		// Print the log of the containers if pod is not running and ready.
 		for _, container := range p.Status.ContainerStatuses {
 			cIdentifer := fmt.Sprintf("%s/%s/%s", p.Namespace, p.Name, container.Name)
-			log, err := framework.GetPodLogs(c, p.Namespace, p.Name, container.Name)
```

Comment:
In some place,when use GetPodLogs that f.Namespace.Name is not same with p.Namespace. It seems that I can't remove the clientset and namespace arguments and get them from f directly. @danwinship @Random-Liu